python/tvm/rpc/tracker.py

```diff
@@ -427,11 +461,14 @@ def terminate(self):
         if self.proc:
             if self.proc.is_alive():
                 self._stop_tracker()
-                self.proc.join(1)
+            self.proc.join(0.1)
```

Comment:
Just curious, why is it changed to 0.1?

Reply:
to make sure the wait time is shorter, since in most cases there are wait